piff/config.py

```diff
@@ -210,36 +207,33 @@ def meanify(config, logger=None):
         verbose = config.get('verbose', 1)
         logger = setup_logger(verbose=verbose)
 
-    for key in ['input', 'output']:
+    for key in ['output', 'hyper']:
         if key not in config:
             raise ValueError("%s field is required in config dict"%key)
     for key in ['file_name']:
-        if key not in config['input']:
+        if key not in config['output']:
             raise ValueError("%s field is required in config dict output"%key)
 
     for key in ['file_name']:
-        if key not in config['output']:
-            raise ValueError("%s field is required in config dict output"%key)
+        if key not in config['hyper']:
+            raise ValueError("%s field is required in config dict hyper"%key)
 
-    if 'dir' in config['input']:
-        dir = config['input']['dir']
+    if 'dir' in config['output']:
+        dir = config['output']['dir']
     else:
         dir = None
 
-    if 'binning' in config['output']:
-        if type(config['output']['binning']) is not int:
-            ValueError("type of binning should be integer (%s now)"%type(config['output']['binning']))
-        else:
-            binning = config['output']['binning']
+    if 'bin_spacing' in config['hyper']:
+        bin_spacing = config['hyper']['bin_spacing'] #in arcsec
     else:
-        binning = 15
+        bin_spacing = 120. #default bin_spacing: 120 arcsec
```

Comment:
20 arcmin seems like a lot.  I'd have guessed more like 5 arcmin maybe? A single CCD in DES is 8x18 arcmin, for reference.  LSST is similar: 14 x 14 arcmin.

Reply:
I guess that I put 120 arcsec per default which are 2 arcmin, but maybe there is something that I am misunderstanding.

Reply:
Oh, that was dumb.  You're right of course.  Somehow I kept thinking that was a factor of 10 larger than it was.  120 as the default is fine.